_data/languages.yml

```diff
@@ -859,7 +921,7 @@
   territories: [ in ]
 
 -
-  codes: [ mn, mon ]
+  codes: [ mn, mon, mo ]
```

Comment:
ditto

Reply:
The issue is that in other contexts this is used for Moldovan.

Reply:
done